tests/parametric/test_config_consistency.py

```diff
@@ -0,0 +1,52 @@
+"""
+Test configuration consistency for functions among different languages for APM.
+"""
+import pytest
+
+from utils import scenarios
+
+parametrize = pytest.mark.parametrize
+
+TEST_SERVICE = "test_service"
+TEST_ENV = "test_env"
+DEFAULT_ENVVARS = {
+    "DD_SERVICE": TEST_SERVICE,
+    "DD_ENV": TEST_ENV,
+}
```

Comment:
I don't think you need these (lines 10-15). Then you can remove `**DEFAULT_ENVVARS` ref on line 20.